lib/msal-browser/docs/acquire-token.md

```diff
@@ -89,10 +79,42 @@ var graphEndpoint = "https://graph.microsoft.com/v1.0/me";
 
 fetch(graphEndpoint, options)
     .then(resp => {
-            //do something with response
+        //do something with response
     });
 ```
 
+## MSAL token acquisition best practices
+
+The following are best practices to acquire tokens with MSAL for avoiding errors, performance hits and usability issues. Certain scenarios may provide exceptions to these.
+
+### Use a single PublicClientApplication instance
+
+Instantiate one `PublicClientApplication` per application and use the same instance throughout your app. This ensures that there is a single source of truth for what MSAL is performing at any given time (see: [MSAL events](events.md)) and eliminates the chance of distinct app objects making parallel interactive requests or potential cache conflicts, which might break apps, reduce performance or hinder user experience.
+
+### Always wait for promises to resolve
+
+All MSAL `acquireToken*` as well as `login*` APIs (which uses `acquireToken*` underneath) perform asynchronous operations and return promises. You should always wait for these promises to resolve, even if you do not need the promise payload.
+
+### Check for interaction status before interactive requests
+
+You should always check for [InteractionStatus](https://azuread.github.io/microsoft-authentication-library-for-js/ref/enums/_azure_msal_browser.interactionstatus.html) being equal to **None** before making an interactive request. `InteractionStatus` will be emitted through [MsalBroadcastService $inProgress observable](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/v2-docs/events.md#the-inprogress-observable) in MSAL Angular and [MsalProvider useMsal() hook](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/docs/hooks.md#usemsal-hook) in MSAL React.
```

Comment:
This is an Angular and React concept - I'd keep this in the docs for those respective libraries

Reply:
Hmm.. I'm thinking this doc as a required reading before msal-angular/react docs, or at least there's benefit in getting familiar with msal-browser token acquisiton APIs even if folks will be working primarily with msal-angular/react. 

Alternatively, do you think we should mention `EventMessageUtils.getInteractionStatusFromEvent` API for obtaining the interaction status (if not using angular or react)

Reply:
Removed this section and added a remark to **Attempt silent request first, then interactive** section for wrapper libraries. Hopefully this would help keeping folks from running into the [issue](https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/5378#discussion_r1019503526) that @1hw7 had